README.md

```diff
@@ -1,141 +1,109 @@
 # Learn ![elixir logo](http://elixir-lang.org/images/logo/logo.png "Elixir Logo")
 
-Learn the Elixir _language_, designed to build dynamic, functional,
-scalable and maintainable web applications!
-
 ## Contents
-- [Why?](#why)
 - [What?](#what)
+- [Why?](#why)
```

Comment:
@Cleop the "_Why_" question is to help understand the person's motivation:
+ Why do I want a particular feature?
+ Why am I feeling pain _not_ having some functionality?

Starting with "_What_" gives the (_potential_) "_solution_" `before` we've _understood_ the challenge/problem the person is facing.